.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - "2.7"
+services: postgresql
+install:
+  - pip install -r requirements.txt
+before_script:
+  - psql -c "CREATE DATABASE vms;" -U postgres
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3
```

Comment:
@tapasweni-pathak xfvb is imitating a display for automated tests. sleep is being used to just give it some time to start